test/system/251-system-service.bats

```diff
@@ -60,6 +60,8 @@ function teardown() {
 
     # %%-remote makes this run real podman even when testing podman-remote
     systemd-run --unit=$SERVICE_NAME ${PODMAN%%-remote*} system service $URL --time=0
+
+    ss -tulpn
```

Comment:
Obviously a debugging artifact. Is this likely to be useful ever again? If so, please add a comment; otherwise please remove.

Reply:
removed